packages/InteractiveIcon/shared/StyledInteractiveIconButton.jsx

```diff
@@ -15,7 +15,13 @@ const getOutline = ({ variant }) => {
 
   return {
     '&:focus': {
-      outline: `0.125rem solid ${colorWhite}`,
+      outline: 'transparent',
+      border: `0.125rem solid ${colorWhite}`,
+      borderRadius: '50%',
+    },
+    '&:active': {
+      backgroundColor: 'rgba(0,0,0,0.5)',
+      backgroundBlendMode: 'multiply',
```

Comment:
`background-blend-mode` is not supported in IE11 or Edge. Will need to keep an eye on this during QA.